packages/uiweb/src/lib/components/widget/subscriptionManager/ManageNotificationComponent.tsx

```diff
@@ -38,19 +35,11 @@ interface IManageNotificationComponentProps {
   userSettings: any;
   handleNext: () => void;
 }
-export const ManageNotficationsComponent: React.FC<
-  IManageNotificationComponentProps
-> = (options: IManageNotificationComponentProps) => {
-  const {
-    userSettings,
-    channelAddress,
-    channelInfo,
-    handleNext,
-    autoconnect = false,
-  } = options || {};
-  const [modifiedSettings, setModifiedSettings] = useState<
-    PushAPI.NotificationSettingType[] | null
-  >([...userSettings]);
+export const ManageNotficationsComponent: React.FC<IManageNotificationComponentProps> = (
+  options: IManageNotificationComponentProps
+) => {
+  const { userSettings, channelAddress, channelInfo, handleNext, autoconnect = false } = options || {};
+  const [modifiedSettings, setModifiedSettings] = useState<PushAPI.NotificationSettingType[] | null>([...userSettings]);
```

Comment:
If the initial state is [...userSettings] do we actually need to set type as null?